### PR TITLE
Changes the PATH variable modified when installing conan package.

### PR DESCRIPTION
**Pull request by Xav83**
Changes the PATH variable modified when installing conan package.

Related to #7

### CODE REVIEW

File: automatic/conan/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -22,4 +22,4 @@ Install-ChocolateyPackage @packageArgs
 
 # Add conan to PATH
 $conan_path = "C:\Program Files\Conan\conan"
-Install-ChocolateyPath -PathToInstall $conan_path -PathType User
+Install-ChocolateyPath -PathToInstall $conan_path -PathType Process
```
Comment (stefansli) [inline automatic/conan/tools/chocolateyInstall.ps1]:
```suggestion
Install-ChocolateyPath -PathToInstall $conan_path -PathType Machine
```
`Process` would mean only the environment of the current executing process and would mean that `conan` isn't available for anyone.

**Review by stefansli — CHANGES_REQUESTED** [review]

**Comment by stefansli** [comment]
I like the approach for the variable to be configurable. But please set the default value to `Machine` since that is what I'd guess most people want. The current default of `Process` doesn't do anything

**Comment by Xav83** [comment]
Actually, I am going to change it back to `User` so that the default behavior of the package won't change for the users who are already installing it. If, in the future, more people asks me to change this behavior, I will re-consider this choice :wink:

**Comment by stefansli** [comment]
Alright, thanks for the update!

**Pull request merged**